test/TestContractHandler.ts

```diff
@@ -98,17 +96,11 @@ export class TestContractHandler {
   }
 
   public async deployContracts(owner: string, routerABI?: AbiItem | AbiItem[]) {
-    const data = JSON.parse(
-      fs.readFileSync(
-        process.env.ADDRESS_FILE ||
-          `${homedir}/.ocean/ocean-contracts/artifacts/address.json`,
```

Comment:
I'm not sure if we should get this out and replaced those addresses since it reads the contract addresses deployed by barge, and I think we should be still using those in our tests. So i think we should focus on replacing endpoints of services like provider Aquarius and web3 for eg in this PR but not the contract addresses

Reply:
nvm just saw the config logic that reads that